samples/retained-instance-store/src/main/java/com/badoo/ribs/samples/retained_instance_store/rib/retainedInstance/routing/RetainedInstanceRouter.kt

```diff
@@ -0,0 +1,35 @@
+package com.badoo.ribs.samples.retained_instance_store.rib.retainedInstance.routing
+
+import android.os.Parcelable
+import com.badoo.ribs.core.modality.BuildParams
+import com.badoo.ribs.routing.Routing
+import com.badoo.ribs.routing.resolution.ChildResolution.Companion.child
+import com.badoo.ribs.routing.resolution.Resolution
+import com.badoo.ribs.routing.router.Router
+import com.badoo.ribs.routing.source.RoutingSource
+import kotlinx.android.parcel.Parcelize
+
+class RetainedInstanceRouter internal constructor(
+        buildParams: BuildParams<Nothing?>,
+        routingSource: RoutingSource<Configuration>,
+        private val builders: RetainedInstanceChildBuilders
+) : Router<RetainedInstanceRouter.Configuration>(
+        buildParams = buildParams,
+        routingSource = routingSource
+) {
+
+    sealed class Configuration : Parcelable {
+            @Parcelize
+            object Retained : Configuration()
```

Comment:
Let's wrap both of them in a sealed class `Content`

Reply:
Done